### PR TITLE
HAWQ-1029. Update hawqregister_help info.

### PR DESCRIPTION
**Pull request by xunzhang**
cc @ictmalili

### CODE REVIEW

**Comment by xunzhang** [comment]
also cc @radarwave

File: tools/doc/hawqregister_help [hunk 1]
```diff
@@ -7,8 +7,8 @@ Usage2: Register parquet/ao table from laterst-sync-metadata in yaml format
 SYNOPSIS
 *****************************************************
 
-Usage1: hawq register [-h hostname] [-p port] [-U username] [-d databasename] [-f filepath] <tablename>
-Usage2: hawq register [-h hostname] [-p port] [-U username] [-d databasename] [-c config] <tablename>
+Usage1: hawq register [-h hostname] [-p port] [-U username] [-d databasename] [-f filepath] [-e eof] <tablename>
+Usage2: hawq register [-h hostname] [-p port] [-U username] [-d databasename] [-c config] [--force] [--repair] <tablename>
```
Comment (ictmalili) [inline tools/doc/hawqregister_help]: Should we add more description about the --force and --repair params?

File: tools/doc/hawqregister_help [hunk 2]
```diff
@@ -100,7 +100,7 @@ Assume the location of the database is 'hdfs://localhost:8020/hawq_default',
 tablespace id is '16385', database id is '16387', table filenode id is '77160',
 last file under the filenode numbered '7'.
 
-$ hawq register postgres parquet_table hdfs://localhost:8020/temp/hive.paq
+$ hawq register -d postgres -f hdfs://localhost:8020/temp/hive.paq parquet_table
```
Comment (ictmalili) [inline tools/doc/hawqregister_help]: +1 for this change

**Comment by xunzhang** [comment]
cc @ictmalili @linwen again

File: tools/doc/hawqregister_help [hunk 3]
```diff
@@ -37,10 +35,19 @@ The file(s) to be registered and the table in HAWQ must be in the
...
+ * If table already exist, hawq register will append the files to the existing table.
+ * If --force option specified, hawq register will erase existing catalog 
+   table pg_aoseg.pg_aoseg_$relid/pg_aoseg.pg_paqseg_$relid data for the table and 
+   re-register according to .yml configuration file definition. Note. If there are
+   files under table directory which are not specified in .yml configuration file, it will throw error out.
+Note. Without --force specified, if some file specified in .yml configuration file lie under the table directory, hawq register will throw error out.
+Note. With --force option specified, if there are files under table directory which are not specified in .yml configuration file, hawq register will throw error out.
+Note. For both the use cases of hawq register, if the table is hash distributed, hawq register just check the file number to be registered has to be integral multiple multiple times of this table’s bucket number, and check whether the distribution key specified in .yml configuration file is same as that of table. It does not check whether files are actually distributed by the key.
```
Comment (ictmalili) [inline tools/doc/hawqregister_help]: Use Case 1 does not support hash distributed table register.

Comment (ictmalili) [inline tools/doc/hawqregister_help]: "hawq register just check the file number to be registered has to be integral multiple multiple times of this table’s bucket number"  There are many useless words there. "integral" and "multiple"

File: tools/doc/hawqregister_help [hunk 4]
```diff
@@ -110,14 +113,24 @@ table 'pg_aoseg.pg_paqseg_77160'.
...
+Secondly, extract the table metadata information out:
+$ hawq extract -o paq1.yml paq1
+
+Thirdly, register to new table paq2 identifying yml file:
+$ hawq register --config paq1.yml paq2
+
+Finally, select the new table to look at whether the content has already been registered.
+$ select count(*) from paq2;
+
+In the above example, the final result must be return 1000.
```
Comment (ictmalili) [inline tools/doc/hawqregister_help]: change "must be" to "should"

File: tools/doc/hawqregister_help (hunk 3 above)
Comment (ictmalili) [inline tools/doc/hawqregister_help]: We

**Comment by xunzhang** [comment]
Merged into master, thanks.

**Pull request closed without merging**